WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40383,918 +40424,371 @@ e_latin_empire = {
 	}
 }
 
-d_miaphysite = {
```

Comment:
I moved it back to where I had removed it, in accordance with _**your**_ orders. It worked fine when I had it in its own file.